cbmc_viewer/symbol_table.py

```diff
@@ -26,24 +35,41 @@ def is_location_line(line):
 
     return line.startswith('Location')
 
+def is_pretty_name_line(line):
+    """Line is a symbol table symbol pretty name line."""
```

Comment:
This documentation is a bit confusing.

Reply:
Revised the documentation of the three is_* functions.